src/browser/api/webcontents.ts

```diff
@@ -0,0 +1,56 @@
+import { WebContents } from 'electron';
+import * as url from 'url';
+
+
+export function executeJavascript(webContents: WebContents, code: string, callback: (e: any, result: any) => void): void {
+    webContents.executeJavaScript(code, true, (result) => {
+        callback(undefined, result);
+    });
+}
+
+export function getAbsolutePath(webContents: WebContents, path: string) {
+    const windowURL = webContents.getURL();
+    return  url.resolve(windowURL, path);
+}
+
+export function getInfo(webContents: WebContents) {
+    return {
+        canNavigateBack: webContents.canGoBack(),
+        canNavigateForward: webContents.canGoForward(),
+        title: webContents.getTitle(),
+        url: webContents.getURL()
+    };
+}
+
+export function getZoomLevel(webContents: WebContents, callback: (zoomLevel: number) => void) {
```

Comment:
can/should we make this function `async` and handle callback here? (thinking about the future...)

Reply:
I'd rather do that in a follow up pr, keep this as close to the original window.js as we can for now